phpunit.xml

```diff
@@ -1,19 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit colors="true" bootstrap="./test/Bootstrap.php">
-	<testsuites>
-		<testsuite name="all">
-			<directory>./test</directory>
-		</testsuite>
-	</testsuites>
+<phpunit colors="true" bootstrap="./vendor/autoload.php">
+    <testsuites>
+        <testsuite name="all">
+            <directory>./test</directory>
+        </testsuite>
+    </testsuites>
 
-	<filter>
-		<whitelist>
+    <filter>
+        <whitelist>
             <directory suffix=".php">src/</directory>
         </whitelist>
-	</filter>
+    </filter>
 
-	<logging>
-        <log type="tap" target="build/report.tap"/>
```

Comment:
Removed this because the `TAP` reporter is deprecated and Coveralls doesn't use it.